trento/xml/article_sap_trento.xml

```diff
@@ -794,7 +786,731 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
     </procedure>
   </section>
 
-  <section xml:id="sec-trento-webconsole">
+
+  <!--
+  Split section sec-trento-webconsole in two sections:
+
+  1. Configuration:
+     Where we describe the different configuration options currently available in the console.
+     Particularly:
+     - cluster settings (we can reutilize the existing content)
+     - tags on overviews (hosts, clusters, sap systems, HANA databases)
+  2. Use cases:
+     dashboard description (we can reutilize the exiting content)
+     - what to do when host status is not green
+     - what to do when pacemaker cluster status is not green
+     - what to do when database status is not green
+     - what to do wen SAP instances status is not green
+  -->
+  <section xml:id="sec-trento-checks">
+    <title>Performing configuration checks</title>
+    <para>
+      One of Trentos main feature is to provide checks for your SAP Landscape.
+      To run these checks, proceed as follows:
+    </para>
+    <procedure xml:id="pro-trento-checks">
+      <title>Performing configuration checks</title>
+      <step>
+        <para>Log in to Trento</para>
+      </step>
+      <step>
+        <para>In the Trento dashboard, click <guimenu>Cluster</guimenu>.</para>
+      </step>
+      <step>
+        <para>In the list, search for a cluster, for example a &hana; cluster.</para>
+      </step>
+      <step>
+        <para>Click the respective cluster name in the name column. The details view opens.</para>
+        <remark>tom 2023-01-16: screenshot</remark>
+        <figure>
+          <title>&pace; cluster details</title>
+          <mediaobject>
+            <imageobject>
+              <imagedata width="70%" fileref="trento-web-pacemaker-cluster-details-view.png"/>
```

Comment:
I accidentally marked the previous comment as resolved but the issue with the screenshot persists:

This screenshot has been take in a small screen. The checks result section should show on the left of the section with the cluster runtime information (provider, SID, fencing type, etc). Both sections should be horizontally aligned.